ports/azure_sdk.md

```diff
@@ -0,0 +1,341 @@
+  /linuxrc
+  ```
+
+  - As You can see, the advanced version of `Phoenix-RTOS` with `POSIX` shell has been started:
+
+  <img src="_images/azure_linuxrc.png" width="700px">
+
+  - Now You can check the internet connection using the `ping` applet.
+
+  - To make the configuration complete please set the current date using the `date` applet, like below:
+
+  <img src="_images/azure_date.png" width="700px">
+
+  - Note: The required argument entered after `@` is EPOCH (POSIX time format) - You can gt it for example from [here](https://www.epochconverter.com/).
+
+
+
+### Running the IoThub Client sample
+
+Providing the internet connection is working and the current date is set, You can run the built `iothub_ll_telemetry_sample`:
+
+```
+/bin/iothub_ll_telemetry_sample
+```
+
+<img src="_images/azure_sample.png" width="700px">
+
+ - Note: The entered connection string was added to the sample's source code, You can find it in `_build/ia32-generic-qemu/azure_sdk/azure-iot-sdk-c-lts_01_2022/iothub_client/samples/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.c`.
+
+You can read the sent messages from Azure, for example using `AzureIotHub VS Code Extension` by clicking `Start Monitoring Built-in Event Endpoint`:
+
+ ```
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+[IoTHubMonitor] [6:07:21 PM] Message received from [ia32-generic-qemu]:
+"test_message"
+```
+
+## Using azure-iot-sdk-c
+
+The above guide only shows how to run one of the provided samples, if You want to write Your own programs using the sdk please follow the next instructions. It may be helpful for the other architectures, like `armv7m7-imxrt106x-evk`, where the generated sample may not work. That's the reason why the following example is adjusted to the configuration with `mbedtls` intended for 'smaller' targets (now only the `imxrt106x` is supported). If You want to write Your own programs intented for `openssl` configuration ('larger' targets, like `ia32-generic-qemu`) there will be a few differences, like no need for setting the certificate in code and increasing stack size in Makefile.
+
+To use functions provided by `azure-iot-sdk-c` please place the specific `azure` libraries (and possibly `mbedtls` libraries) in `LIBS` variable in `Makefile` and include the required header file. Below is the example of an elementary program similar to `iothub_client_ll_telemetry_sample`, You can place it for example in the `_user` directory:
+
+ - Note: The SDK was tested for the `armv7m7-imxrt106x` target architecture with the `mbedtls` port, but remember that internet connection and current date have to be also provided.
+
+ - Makefile - linking with azure SDK and mbedtls libraries.
```

Comment:
```suggestion
 - Makefile - linking with Azure SDK and mbedtls libraries.
```
I don't know, how `Azure` should be spelled - starting from a capital letter or a small letter, but please be consistent with it.